bokeh/models/mappers.py

```diff
@@ -80,3 +92,13 @@ class LogColorMapper(ColorMapper):
         non-negative.
 
     """
+
+    low = Float(help="""
+    The minimum value of the range to map into the palette. Values below
+    this are clamped to ``low``.
+    """)
+
+    high = Float(help="""
+    The maximum value of the range to map into the palette. Values above
+    this are clamped to ``high``.
+    """)
```

Comment:
We should probably factor out a `ContinuousColorMapper` for linear and log mappers to inherit from.